src/Infra/Infra.Twitch/TwitchConnection.cs

```diff
@@ -45,25 +45,25 @@ public void Disconnect()
             throw new NotImplementedException();
         }
 
-        protected virtual void Client_OnLog(object sender, OnLogArgs e)
+        protected virtual void Client_OnLog(object? sender, OnLogArgs e)
```

Comment:
All these sender args need to be nullable since twitchlib is not c#8

Reply:
*yet

I'll ping LuckyNoS7evin to see if his restructure is going to change that.